packages/api-nest/src/repositories/highlight.repository.ts

```
@@ -109,4 +109,43 @@ export class HighlightRepository implements IHighlightRepository {
 
     return result
   }
+
+  /**
+   * Find all highlights for a user with pagination
+   * Sorted by creation date (most recent first)
+   */
```

ESLint

[error] Missing JSDoc @returns for function.

(valid-jsdoc)